stm32-modules/flex-stacker/firmware/motor_control/motor_hardware.c

```diff
@@ -305,19 +307,20 @@ void hw_enable_motor(MotorID motor_id) {
             port = L_EN_PORT;
             pin = L_EN_PIN;
             status = HAL_TIM_Base_Start_IT(&htim3);
-            HAL_NVIC_SetPriority(TIM3_IRQn, 6, 0);
+            HAL_NVIC_SetPriority(TIM3_IRQn, 10, 0);
             HAL_NVIC_EnableIRQ(TIM3_IRQn);
             if (status == HAL_OK)
             {
             }
             break;
         default:
-            return;
+            return status == HAL_OK;
```

Comment:
This default case should probably explicitly return `false`, since we did not match any of the cases and thus did not `enable` a motor.